Added PowerShell

bastianeicher opened the pull request:
`index.html` for the `powershell` directory:

```html
<!DOCTYPE html>
<html>
  <head><title>repo.roscidus.com: PowerShell</title></head>
  <body>
    <h1>repo.roscidus.com: PowerShell</h1>

    <h2>Available feeds</h2>
    <ul>
      <li><a href="http://repo.roscidus.com/powershell/powershell">PowerShell (Windows or Core)</a></li>
      <li><a href="http://repo.roscidus.com/powershell/core">PowerShell Core</a></li>
      <li><a href="http://repo.roscidus.com/powershell/windows">Windows PowerShell</a></li>
    </ul>

    <h2>Samples</h2>

    <p>To use PowerShell as the runner in a feed:</p>
    <pre>      &lt;command name="run" path="myscript.ps1"&gt;
        &lt;runner interface="http://repo.roscidus.com/powershell/powershell"&gt;
          &lt;arg&gt;-ExecutionPolicy&lt;/arg&gt;
          &lt;arg&gt;Bypass&lt;/arg&gt;
        &lt;/runner&gt;
      &lt;/command&gt;
    </pre>

    <p>To add <code>pwsh</code> (PowerShell Core) to your <code>PATH</code>:</p>
    <pre>      $ 0alias pwsh http://repo.roscidus.com/powershell/core</pre>
  </body>
</html>
```

talex5 commented:
```
$ 0repo
Feed 'feeds/powershell/windows.xml' with uri='http://repo.roscidus.com/powershell/wmf' should be located at 'feeds/powershell/wmf.xml'
```

bastianeicher commented:
Ah, sorry about that. I had originally intended on calling that feed `powershell/wmf` (for _Windows Management Framework_)but decided that `powershell/windows` (for _Windows PowerShell_, the actual product name as it's shown in the OS) would be more intuitive.  
I fixed the URI in the feed file and rebased the branch on the current master.

Thanks for all the maintainer work. :) I wish you happy holidays and a happy new year.

talex5 commented:
Thanks!
